cmd/compose/watch.go

```diff
@@ -44,6 +44,10 @@ func watchCommand(p *ProjectOptions, backend api.Service) *cobra.Command {
 			return runWatch(ctx, backend, opts, args)
 		}),
 		ValidArgsFunction: completeServiceNames(p),
+		Annotations: map[string]string{
```

Comment:
If "watch" is a subcommand of "alpha", then _probably_ this isn't needed (but double check if I'm right); IIRC, we make the YAMLdocs check if parents are experimental, and make child-commands inherit that.